src/main/java/jenkins/plugins/logstash/persistence/LogstashIndexerDao.java

```diff
@@ -37,7 +37,7 @@
  * @since 1.0.0
  */
 public interface LogstashIndexerDao {
-  static enum IndexerType {
+  public static enum IndexerType {
```

Comment:
interface members are public by default, I'd prefer to not restate default modifiers

Reply:
ok